File: codingame/golf/power_of_thor.go
/* Simple
package main
import "fmt"
func main() {
    var x, y, h, v, t int
    fmt.Scanf("%d%d%d%d", &x, &y, &h, &v)
    for {
       fmt.Scanf("%d",&t)
       if (y<v) {fmt.Printf("N");v-=1}
       if (y>v) {fmt.Printf("S");v+=1}
       if (x<h) {fmt.Printf("W");h-=1}
       if (x>h) {fmt.Printf("E");h+=1}
       fmt.Println()
    }
}

## 249
package main
import "fmt"
func main(){
var x,y,h,v,t int
fmt.Scanf("%d%d%d%d",&x,&y,&h,&v)
for{fmt.Scanf("%d",&t)
if(y<v){fmt.Printf("N");v-=1}
if(y>v){fmt.Printf("S");v+=1}
if(x<h){fmt.Printf("W");h-=1}
if(x>h){fmt.Printf("E");h+=1}
fmt.Println()}}
*/

File: codingame/golf/power_of_thor.cpp
/* Simple
#include <iostream>
#define i std::cin
#define p std::cout
int main()
{
    int x,y,h,v,t;
    i >> x >> y >> h >> v;
    for(;;) {
       i >> t;
       if (y<v) {p << "N";v-=1;}
       if (y>v) {p << "S";v+=1;}
       if (x<h) {p << "W";h-=1;}
       if (x>h) {p << "E";h+=1;}
       p << "\n";
    }
}

# Simple 205

#include <iostream>
#define i std::cin
#define p std::cout
int main(){int x,y,h,v,t;i>>x>>y>>h>>v;for(;;){i>>t;if(y<v){p<<"N";v-=1;}if(y>v){p<<"S";v+=1;}if(x<h){p<<"W";h-=1;}if(x>h){p<<"E";h+=1;}p<<"\n";}}
*/


#include <iostream>
#define i std::cin
#define p std::cout
int main(){int x,y,h,v,t;i>>x>>y>>h>>v;for(;;){i>>t;if(y<v){p<<"N";v-=1;}if(y>v){p<<"S";v+=1;}if(x<h){p<<"W";h-=1;}if(x>h){p<<"E";h+=1;}p<<"\n";}}